src/controllers/Tweets.js

```diff
@@ -31,17 +31,18 @@ const getTweetsByUsername = async (req, res) => {
       max_results: limit ? Math.min(limit, 10) : 10, // max 10 tweets
     });
 
-    // TODO: get tweet ids from DB, compare, etc.
-    const screenshots = await getAllTweetScreenshots(
-      username,
-      tweets.map(({ id }) => id)
-    );
+    if (JSON.parse(saveTweets)) {
```

Comment:
I added this functionality to stop saving screenshots every time the endpoint is hit (testing, etc)
It's a query param `saveTweets=true` will allow saving screenshots